### PR TITLE
feat: allow getMultipleAccountsInfo to handle 100+ requests

### PR DESCRIPTION
**Pull request by marcnjaramillo**
#### Problem
The `getMultipleAccountsInfo` RPC call  can only handle 100 accounts at a time. In order to get data on more than 100 accounts, a developer would need to make separate requests. Ideally, we could allow for more than 100 accounts and have the function batch them to the RPC in groups of up to 100.

#### Summary of Changes
- Create a 2-D array consisting of the incoming keys array split into chunks of 100.
- Iterate over the 2-D array, store each result value in an `accounts` array, return once completed.

### CODE REVIEW

File: web3.js/src/connection.ts [hunk 1]
```diff
@@ -2557,31 +2558,60 @@ export class Connection {
...
-        encoding = configOrCommitment.encoding || 'base64';
+    let accounts : (AccountInfo<Buffer | ParsedAccountData> | null)[] = [];
+    let count = 0;
+    let maxData = 0;
+
+    for (let i = 0; i < chunks.length; i++) {
+
+      // this enforces a 10 second pause periodically to ensure rate limits aren't exceeded
+      if (count === 25) {
+        await sleep(10000);
```
Comment (mvines) [inline web3.js/src/connection.ts:2571]: Explicit sleeps like this should be avoided.  The RPC servers will respond with 429 with a [Retry-After](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After) header indicating the client must throttle

Comment (marcnjaramillo) [reply web3.js/src/connection.ts:2571]: Thanks for the feedback. I'll remove the explicit sleep calls and re-submit. Still new to the Solana ecosystem, so I appreciate this being pointed out!

Comment (marcnjaramillo) [reply web3.js/src/connection.ts]: Removed the sleep calls and adjusted the size of chunks. Testing locally everything works. Looking forward to your feedback.

**Comment by marcnjaramillo** [comment]
Due to changes that have recently been merged, this solution no longer works. I'm closing this PR so I can fix it.

**Pull request closed without merging**